mxnet/inference/docker/1.9/py3/cu112/Dockerfile.gpu

```diff
@@ -0,0 +1,116 @@
+FROM nvidia/cuda:11.2.1-cudnn8-runtime-ubuntu20.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+# Specify accept-bind-to-port LABEL for inference pipelines to use SAGEMAKER_BIND_TO_PORT
+# https://docs.aws.amazon.com/sagemaker/latest/dg/inference-pipeline-real-time.html
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+ARG MMS_VERSION=1.1.2
+ARG MX_URL=https://aws-mx-pypi.s3-us-west-2.amazonaws.com/1.9.0/aws_mx_cu112-1.9.0-py3-none-manylinux2014_x86_64.whl
+
+ENV PYTHONDONTWRITEBYTECODE=1 \
+    PYTHONUNBUFFERED=1 \
+    LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/usr/local/lib" \
+    PYTHONIOENCODING=UTF-8 \
+    LANG=C.UTF-8 \
+    LC_ALL=C.UTF-8 \
+    TEMP=/home/model-server/tmp \
+    DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update \
+ && apt-get -y install --no-install-recommends \
+    build-essential \
+    ca-certificates \
+    curl \
+    emacs \
+    git \
+    libopencv-dev \
+    openjdk-8-jdk-headless \
+    openjdk-8-jdk \
+    openjdk-8-jre \
+    vim \
+    wget \
+    unzip \
+    zlib1g-dev \
+    libreadline-gplv2-dev \
+    libncursesw5-dev \
+    libssl-dev \
+    libsqlite3-dev \
+    tk-dev \
+    libgdbm-dev \
+    libc6-dev \
+    libbz2-dev \
+    libffi-dev \
+    openssl \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+# install Python
+RUN apt-get update \
+ && apt-get install -y software-properties-common \
+ && add-apt-repository -y ppa:deadsnakes/ppa \
```

Comment:
why deadsnakes is needed? python3.8 is already installed on UL20

Reply:
`apt-get install python3` gives v3.8.2 https://packages.ubuntu.com/focal/python3
whereas deadsnakes gives the latest v3.8.11 https://github.com/deadsnakes/python3.8/blob/upstream/README.rst
So there might be some fixes made in the latest version.

Reply:
But I dont see any issues with v3.8.2 as of now, so will use `apt-get`